R/db-sql.R

```diff
@@ -219,15 +231,16 @@ sql_query_select.DBIConnection <- function(con, select, from, where = NULL,
                                limit = NULL,
                                distinct = FALSE,
                                ...,
-                               subquery = FALSE) {
+                               subquery = FALSE,
+                               lvl = 0) {
```

Comment:
It might be nicer to apply apply the level indent in `sql_select_clauses()`. Consequences:

* no need to pass `lvl` to `sql_clause_*()`
* `sql_clause_*()` would need to return a character vector instead of a string
* `sql_select_clauses()` would need to get a `lvl` argument